scripts/common/preflights.sh

```diff
@@ -657,3 +657,19 @@ function bail_if_unsupported_openebs_to_rook_version() {
     fi
 }
 
+# bail_when_no_object_store_and_s3_enabled will bail if Minio and Rook are not present and kotsadm.s3Disabled is false.
+function bail_when_no_object_store_and_s3_enabled() {
+    if [[ -z "$MINIO_VERSION" && -z "$ROOK_VERSION" ]] && [ -n "$OPENEBS_VERSION" ]; then
```

Comment:
Openebs is not important here

```suggestion
    if [ -z "$MINIO_VERSION" ] && [ -z "$ROOK_VERSION" ]; then
```